tests/queries/0_stateless/02354_parse_timedelta.sql

```diff
@@ -6,7 +6,8 @@ SELECT parseTimeDelta('1yr2mo');
 SELECT parseTimeDelta('11s+22min');
 SELECT parseTimeDelta('1yr-2mo-4w + 12 days, 3 hours : 1 minute ; 33 seconds');
 SELECT parseTimeDelta('1s1ms1us1ns');
-SELECT parseTimeDelta('1s1ms1μs1ns');
+SELECT parseTimeDelta('1s1ms1μs1ns'); // μs U+03BC
+SELECT parseTimeDelta('1s1ms1µs1ns'); // μs U+00B5
```

Comment:
The comment looks wrong:

```
$ curl --location https://github.com/ClickHouse/ClickHouse/pull/75472.diff | cat -A
-        {"M-NM-<s", 1e-6},$
+        {"M-NM-<s", 1e-6}, // U+03BC = Greek letter mu$
+        {"M-BM-5s", 1e-6}, // U+00B5 = micro symbol$
...
-SELECT parseTimeDelta('1s1ms1M-NM-<s1ns');$
+SELECT parseTimeDelta('1s1ms1M-NM-<s1ns'); // M-NM-<s U+03BC$
+SELECT parseTimeDelta('1s1ms1M-BM-5s1ns'); // M-BM-5s U+00B5$
```

The last line should be:

```
+SELECT parseTimeDelta('1s1ms1M-BM-5s1ns'); // M-NM-<s U+00B5$
```

```suggestion
SELECT parseTimeDelta('1s1ms1µs1ns'); // µs U+00B5
```

Let's see how commit via github UI will work.